### ISSUE TITLE
evaluation classes

### ISSUE THREAD

**Issue by ryosism** [epic-kitchens/annotations#14]
Hi,

thank you for grate dataset, I'd like to eval my trained model with your test data.
I know EPIC-Kitchens Object Detection challenge has submission form, but I don't know what classes should be detected.

Should my model detect all noun classes(784 classes) or 15 most frequent object classes?

thanks.

**Comment by hazeld** [comment]
Hi ryosism,

Thank you for your interest in the dataset. Your model should detect all **clustered** noun classes which appear in the training set (of which there are 295). You can find the noun classes for each bounding box in the `noun_class` column of the `EPIC_train_object_labels.csv`. The class label and names of objects contained in each class can be found in `EPIC_noun_classes.csv`.

**Comment by ryosism** [comment]
Thank you! It is helpful for me.
I think there are some classes isn't appeared in `EPIC_train_object_labels.csv` (e.g. 149 "butter:peanut"), Is any reason why these classes are included in `EPIC_noun_classes.csv`?

**Comment by hazeld** [comment]
Yes, there will be classes that appear in `EPIC_noun_class.csv` that aren't in `EPIC_train_object_labels.csv` as we have noun classes which appear in the test set but not in training. These zero shot classes aren't tested for in the object detection challenge.
